modules/dmrpp_module/CurlHandlePool.cc

```diff
@@ -242,10 +242,17 @@ dmrpp_easy_handle::~dmrpp_easy_handle() {
 #if NEW_WAY
```

Comment:
If we're going to commit to doing this the NEW_WAY then lets drop the #if conditional...

Reply:
Done